app/react/translations/edit/editors/HTMLEditor/index.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import React from 'react';
+
+import RawEditor from './RawEditor';
+import RichEditor from './RichEditor';
+
+type PropTypes = {
+  onSubmit: Function,
+  // eslint-disable-next-line react/no-unused-prop-types
+  onChange: Function,
```

Comment:
Is it needed for proper functionality of component? Otherwise, it should be removed.